test-tools/src/test/java/org/terracotta/org/junit/rules/TemporaryFolderUsageTest.java

```diff
@@ -98,6 +99,8 @@ public void newFolderWithGivenFolderThrowsIOExceptionIfFileExists() throws IOExc
 
     @Test
     public void newFolderWithGivenFolderThrowsIOExceptionWhenFolderCannotBeCreated() throws IOException {
+        assumeFalse(Boolean.getBoolean("inside-container"));
```

Comment:
This use case is not supported inside Docker.
Probably `setReadOnly()` has no effect ?